worldedit-core/src/main/java/com/fastasyncworldedit/core/util/EditSessionBuilder.java

```diff
@@ -43,14 +43,17 @@
 import com.sk89q.worldedit.util.formatting.text.TextComponent;
 import com.sk89q.worldedit.world.World;
 import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.Nullable;
 
 import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
 import java.util.Locale;
 import java.util.UUID;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+
+//TODO Migrate all logic to com.sk89q.worldedit.EditSessionBuilder
```

Comment:
Is the TODO still up to date? Checking out the other class we already got this covered.